CONTRIBUTING.md

```diff
@@ -137,11 +137,13 @@ Currently the process of releasing a new version of Langium is done manually one
     * Update version in `package.json`
     * Update dependency to `langium`
     * `npm install -g vsce`
```

Comment:
I just saw that the package has been renamed to `@vscode/vsce`: https://www.npmjs.com/package/vsce